app/controllers/borrow_requests_controller.rb

```diff
@@ -6,7 +6,9 @@ def index
                                             .ordered_by_status
   end
 
-  def show; end
+  def show
+    @borrow_req = BorrowRequest.find_by id: params[:id]
```

Comment:
thêm code vào nào, xử lý nil ne, xử lý xem có đúng request của  current user ko nũa nhé 
```suggestion
    @borrow_req = @current_user.borrow_requests.find_by id: params[:id]
```

Reply:
fixed